components/ImageBanner/src/index.scss

```diff
@@ -15,125 +15,126 @@
 }
 
 .denhaag-image-banner__card {
-  align-items: var(--denhaag-image-banner-card-align-items);
-  display: var(--denhaag-image-banner-card-display);
-  height: var(--denhaag-image-banner-card-height);
-  overflow: var(--denhaag-image-banner-card-overflow);
-  position: var(--denhaag-image-banner-card-position);
+  align-items: var(--denhaag-image-banner-card-align-items, end);
+  display: flex;
+  height: var(--denhaag-image-banner-card-height, 378px);
+  overflow: var(--denhaag-image-banner-card-overflow, hidden);
+  position: var(--denhaag-image-banner-card-position, relative);
 }
 
 .denhaag-image-banner__card--focus,
 .denhaag-image-banner__card:focus-visible {
-  outline: var(--denhaag-image-banner-card-focus-outline);
+  outline: var(--denhaag-image-banner-card-focus-outline, var(--denhaag-focus-border));
 }
 
 .denhaag-image-banner__image {
-  height: var(--denhaag-image-banner-image-height);
-  left: var(--denhaag-image-banner-image-left);
-  top: var(--denhaag-image-banner-image-top);
-  object-fit: var(--denhaag-image-banner-image-object-fit);
-  position: var(--denhaag-image-banner-image-position);
-  transition: var(--denhaag-image-banner-transition);
-  width: var(--denhaag-image-banner-image-width);
-  z-index: var(--denhaag-image-banner-image-z-index);
+  height: var(--denhaag-image-banner-image-height, 100%);
+  left: var(--denhaag-image-banner-image-left, 0);
+  top: var(--denhaag-image-banner-image-top, 0);
+  object-fit: var(--denhaag-image-banner-image-object-fit, cover);
+  position: var(--denhaag-image-banner-image-position, absolute);
```

Comment:
Lijkt erop dat dit altijd `absolute` zal blijven, haal de hele CSS variabel maar weg :D zelfde geld voor `object-fit`, `height`, `width`, `top`, `left,`right` en `bottom` dat is iets wat je hier wilt hebben.